students/uss_tomasz/lesson_09_reading_and_writing_files/multiclipboard/msb.pyw

```diff
@@ -0,0 +1,34 @@
+# msb.pyw - saves and loads pieces of text to the clipboard.
+# usage: py.exe msb.pyw save <keyword> - saves clipboard to keyword.
+#        py.exe mcb.pyw <keyword> - loads keyword to clipboard.
+#        py.exe mcb.pyw list - loads all keywords to clipboard.
+#        py.exe mcb.pyw delete <keyword> - delete keyword from shelve
+#        py.exe mcb.pyw delete - delete all content from shelve
+
+import pyperclip
+import shelve
+import sys
+mcb_shelf = shelve.open('mcb')
+if len(sys.argv) == 3:
```

Comment:
Args count should not top-level condition. It'd be much easier to follow program flow if you reorganized the code in a way of switch-case pattern:
```
command = ...get command string
if command == 'save':
  ...handle save
elif command == 'delete':
   ...handle delete
...
else:
  ...handle wrong command
```

In current implementation e.g. delete command is split into 2 places.